README.md

```diff
@@ -61,6 +61,73 @@ To exit the SSH session, type `exit` or type the ssh escape sequence `~.` . (See
 the [ssh man page](https://linux.die.net/man/1/ssh) for other escape sequences).
 Typing `Ctrl+D` or `logoff` at the IEx prompt to exit the session won't work.
 
+## Erlang distribution
+
+`nerves_pack` does not start Erlang distribution. Distribution is not hard to
+enable, but it requires some thought on node naming and security.
+
+Erlang distribution requires that the hostname part of the device's node name be
+reachable from the computer that's trying to connect. Options include IP
+addresses, DNS names, mDNS names or names that you put in your `/etc/hosts`
+file. Many Nerves users use mDNS names for simplicity, but they have
+limitations. You may need to adjust the following script based on your
+environment.
+
+The Nerves project generator configures
+[`mdns_lite`](https://github.com/pcmarks/mdns_lite) to advertise two hostnames:
+`nerves.local` and `nerves-1234.local`. The latter one is based on the serial
+number of the device. If you only have one Nerves device on the network, use
+`nerves.local`. If you have many devices, you'll have to figure out the name
+with the serial number. This can be done by using a mDNS discovery program or by
+logging into a device via a serial console and typing `hostname` at the IEx
+prompt.
+
+The following uses `nerves.local`, but substitute for the name that you want.
+Run this by `ssh`'ing into your Nerves device.
+
+```elixir
+iex> System.cmd("epmd", ["-daemon"])
+{"", 0}
+iex> Node.start(:"nerves@nerves.local")
+{:ok, #PID<0.26318.2>}
+iex(nerves@nerves.local)> :erlang.set_cookie(node(), :my_secret_cookie)
```

Comment:
This can also be `Node.set_cookie(:my_secret_cookie)`

Reply:
I'm still used to the old ways. That's better. Thanks!